apps/settings/style/icons.css

```diff
@@ -293,8 +293,9 @@ aside.connecting {
  */
 
 html[dir="rtl"] a.menu-item:after {
-  left: 0.3rem;
+  left: -0.9rem;
   right: unset;
+  transform: scaleX(-1);
```

Comment:
Let’s use the “back” icon instead of mirroring the “forward” one (much better perfs when panning/scrolling):
```css
  content: "back-light";
```